renovate.json5

```diff
@@ -0,0 +1,94 @@
+{
+  "extends": [
+    "github>defenseunicorns/uds-common//config/renovate.json5",
+    ":semanticCommits",
+    ":semanticCommitTypeAll(chore)"
+  ],
+  "ignorePresets": [
+    ":ignoreModulesAndTests"
+  ],
+  "branchConcurrentLimit": 0,
+  "prConcurrentLimit": 0,
+  "prHourlyLimit": 0,
+  "packageRules": [
+    {
+      "matchManagers": ["npm"],
+      "matchFileNames": ["package-lock.json"],
+      "groupName": "All npm dependencies",
+      "groupSlug": "npm-dependencies"
+    },
+    {
+      "enabled": false,
+      "matchFileNames": ["src/content/docs/structure/packages.md"]
+    },
+    // Keep podinfo chart version + container image tag updates in a single PR so docs never reference mismatched versions.
+    {
+      "groupName": "podinfo docs",
+      "groupSlug": "podinfo-docs",
+      "matchManagers": ["custom.regex"],
+      "matchDepNames": [
+        "https://github.com/stefanprodan/podinfo.git",
+        "ghcr.io/stefanprodan/podinfo"
+      ]
+    }
+  ],
+  "customManagers": [
+    // Matches a ghcr.io repository (including oci://) and captures the next ref version, even if it spans multiple package blocks.
+    {
+      "customType": "regex",
+      "managerFilePatterns": [
+        "/^src\\/content\\/docs\\/.*\\.(md|mdx)$/"
+      ],
+      // Test: https://regex101.com/r/2bifNc/
+      "matchStrings": [
+        "repository:\\s*(?:(?:oci://)?(?<registry>ghcr\\.io)/(?<depPath>[^\\s]+))[\\s\\S]*?ref:\\s*(?<currentValue>[^\\s\"']+)"
+      ],
+      "datasourceTemplate": "docker",
+      "depNameTemplate": "{{registry}}/{{depPath}}",
+      "versioningTemplate": "regex:^v?(?<major>\\d+)(\\.(?<minor>\\d+))?(\\.(?<patch>\\d+))?",
+      "extractVersionTemplate": "^(?<version>v?\d+\.\d+(?:\.\d+)?)(?<suffix>-(?:uds\.\d+(?:-[\w.-]+)?|[\w.-]+))?$",
+      "autoReplaceStringTemplate": "{{{newValue}}}{{{suffix}}}",
```

Comment:
Are there specific dependencies this is meant to support? vs just using the default docker versioning?